### DISCUSSION TITLE
Add a behavior annotation feature

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#989] [Ideas]
We currently use BORIS (http://www.boris.unito.it/pages/boris_v8.html) to manually annotate mouse behaviors.  It would be valuable to also have this feature included in SLEAP. This would cut the work in half for labs that both manually annotate behaviors as well as track animal skeletons.

Implementation would require a window in the GUI that allows you to assign a label to n number of designated frames (for both state behaviors that have start/stop flags and point behaviors that occur only within the current frame).

**Comment by talmo** [comment]
Hi @olinesn,

This has been requested before in #785 -- check out that thread for some suggestions on what else you could use for this task.

That said, I agree that it would be nice to have everything in a single GUI so you can add annotations while labeling poses as well.

Talmo

**Reply by olinesn** [reply]
Thanks for the heads up, comment upvoted!
